src/components/modules/api/toolbar.ts

```diff
@@ -15,6 +15,7 @@ export default class ToolbarAPI extends Module {
     return {
       close: (): void => this.close(),
       open: (): void => this.open(),
+      openToolbox: (): void => this.openToolbox(),
```

Comment:
maybe `toogleToolbox(openingState?: boolean)` will be more convenient, like `toggleBlockSettins`? It will allow closing the Toolbox without closing the Toolbar